frontend/src/components/Metrics/CustomMetrics.tsx

```diff
@@ -264,6 +264,8 @@ class CustomMetrics extends React.Component<Props, MetricsState> {
                   direction={this.state.dashboard?.title || 'dashboard'}
                   labelsSettings={this.state.labelsSettings}
                   hasHistograms={hasHistograms}
+                  hasHistogramsAverage={hasHistograms}
+                  hasHistogramsPercentiles={hasHistograms}
```

Comment:
I see this looks different than frontend/src/components/Metrics/IstioMetrics.tsx on lines 333-4.. which has:
```
                hasHistogramsAverage={hasHistogramsAverage}
                hasHistogramsPercentiles={hasHistogramsPercentiles}
```
But the code here uses `hasHistograms` for all the keys `hasHistograms`, `hasHistogramsAverage`, and `hasHistogramsPercentiles`... is this expected? Or should u use the same variable names as I show above from the other place in the code that does this?

Reply:
Right, it's correct as-is.  CustomMetrics.tsx is just satisfying the need for new properties.  I could have made them optional properties and defaulted to true, but I wanted to make sure they got set where appropriate.  The metrics we support for crippling are the standard istio histograms, not anything custom.  So, if a custom histogram exists we assume it's in tact, and so we can use the same flag for all three props.